recipes/ants/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+if errorlevel 1 exit 1
+cd build
+if errorlevel 1 exit 1
+
+cmake -G Ninja ^
+    %CMAKE_ARGS% ^
+    -DCMAKE_INSTALL_LIBDIR:STRING=%LIBRARY_LIB% ^
+    -DCMAKE_INSTALL_PREFIX:STRING=%LIBRARY_PREFIX% ^
+    -DCMAKE_PREFIX_PATH:STRING=%LIBRARY_PREFIX% ^
+    -DBUILD_SHARED_LIBS:BOOL=ON ^
+    -DANTS_SUPERBUILD:BOOL=OFF ^
+    -DUSE_SYSTEM_ITK:BOOL=ON ^
+    %SRC_DIR%
+if errorlevel 1 exit 1
+
+cmake --build . --config RelWithDebInfo --parallel %CPU_COUNT% --target install
+if errorlevel 1 exit 1
```

Comment:
Why are you skipping Windows? Also, if you are skipping, you don't need this file.

Reply:
There is a build issue on Windows but it's unclear whether this issue is [upstream](https://github.com/ANTsX/ANTs/issues/1547) or from the [ITK conda package](https://github.com/conda-forge/itk-feedstock/issues/37). I left the recipe for Windows so that either of these upstream maintainers could try the build by themselves.